bin/test-grammars

```diff
@@ -33,13 +33,15 @@ TODO_TEST = frozenset((
     'source.tsx',
     'source.x86',
     'source.x86_64',
+    'source.haskell',
```

Comment:
new grammars should have tests

Reply:
Oh, sorry, i didn't had understand it in this moment. Going to fix it.

Reply:
I tried to added `testdata/source_haskell.hs` as a testcase, I update the syntax, I download the syntax, I do `pip install . babi`, but i always get the following result:
![babi_haskell](https://user-images.githubusercontent.com/94523763/149679774-f58f6126-45a9-42c0-abec-00a4287d0b01.png)

Reply:
do you have a theme installed?

Reply:
yeah, the standard vs-code-dark theme. Normally it look like this
![babi_normal](https://user-images.githubusercontent.com/94523763/149680286-215a37fa-9646-41d2-879d-6b5c9eddb444.png)
:

Reply:
`hash -r`? maybe it's using another `babi` on your system?

Reply:
Yep, was the false babi, it had used the global installed babi, when it should have used the babi installed in the virtualenv. Thanks for helping me with this,